### PR TITLE
Link interesting tests in the `/example` README file

### PR DESCRIPTION
**Pull request by amityadav0**
## Proposed Changes

+ Link interesting tests in the `/example` README file
+ Bug fix - add zod in runtime #463

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -60,3 +60,10 @@ the following positions in the latter file:
 - [JS] [signed-txn](./signed-txn) - demonstrates loading a serialized transaction from a file. A user can create a transaction and sign it (using `goal`) and send it to someone else to execute it in `algob`.
 - [TS] [htlc](./htlc-pyteal-ts) - Hash-Time-Lock-Contract Example using PyTeal.
   In this project we are transpiling the files in `js` and then using them with `algob`.
+
+### Tests Description
```
Comment (ratik21) [inline examples/README.md:64]: we should update the heading to something else. Maybe "Few interesting integration tests"?

**Review by ratik21 — COMMENTED** [review]
Please add an entry to the changelog regarding bug fix.

File: examples/README.md (hunk 1 above)
Comment (robert-zaremba) [reply examples/README.md:64]:
```suggestion
### Tests Description
```
```suggestion
### Interesting Test Suites
```

**Review by robert-zaremba — APPROVED** [review]
pre approving.
Please link to this README file section in https://github.com/scale-it/algo-builder/blob/master/docs/guide/testing-teal.md

**Pull request merged**